src/main/java/core/basesyntax/service/strategy/impl/BalanceOperation.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.strategy.impl;
+
+import core.basesyntax.dao.StorageDaoImpl;
+import core.basesyntax.dto.FruitTransactionDto;
+import core.basesyntax.service.strategy.OperationHandler;
+import java.util.HashMap;
+
+public class BalanceOperation implements OperationHandler {
+    private StorageDaoImpl actionDB = new StorageDaoImpl();
```

Comment:
For all handlers, you're creating a new `StorageDaoImpl();`. Here, you can also simply make a field and pass a reference for the DAO as a parameter to the constructor.

Reply:
This way u will create ur DAO once and use the same dao in all handlers.